pgjdbc/src/main/java/org/postgresql/core/v3/QueryExecutorImpl.java

```diff
@@ -2123,8 +2121,8 @@ protected void processResults(ResultHandler handler, int flags) throws IOExcepti
           if (currentPortal != null) {
             currentPortal.close();
           }
-          break;
         }
+          break;
```

Comment:
Is this line motion required?
I think it looks better when `break` is within the braces.